scripts/linux/install.sh

```diff
@@ -10,6 +10,12 @@ then
     exit -1
 fi
 
+if [ -n "$DSN_ROOT" -a -f "$DSN_ROOT/lib/libdsn.core.so" ]
```

Comment:
But make copy action totally in install step is not convenient, I think just leave it as now is OK.